src/commands/init.command.ts

```diff
@@ -155,27 +156,29 @@ const renameGitignoreFiles = async (workspacePath: string) => {
   );
 };
 
-// Needed until https://github.com/unjs/nypm/issues/115 is resolved
-const pmToInstallCommandMap: Record<PackageManagerName, [string, string[]]> = {
-  npm: ['npm', ['ci']],
-  yarn: ['yarn', ['install', '--frozen-lockfile']],
-  bun: ['bun', ['install', '--frozen-lockfile']],
-  pnpm: ['pnpm', ['install', '--frozen-lockfile']],
+type PMTypePromptResult = Pick<PackageManager, 'name'>;
+const promptForPackageManagerSelect = async (): Promise<PMTypePromptResult> => {
+  const temp = await Enquirer.prompt<PMTypePromptResult>({
+    type: 'select',
+    name: 'name' satisfies keyof PMTypePromptResult,
+    message: "Select the package manager you'd like to use:",
+    choices: ['npm', 'bun', 'pnpm', 'yarn'] satisfies PackageManagerName[],
+    // This is to workaround a bug
+    // https://github.com/enquirer/enquirer/issues/121
+    result(choice) {
+      return (this as any).map(choice)[choice];
+    },
+  });
+  return temp;
 };
 
 const installDependenciesWithMessage = async (workspacePath: string) => {
+  const pm = await promptForPackageManagerSelect();
   const spinner = ora('Installing dependencies').start();
-  const pm = await detectPackageManager(workspacePath);
-  if (!pm) {
-    throw new Error(
-      'No package manager found in the workspace. Unable to install dependencies.',
-    );
-  }
```

Comment:
I removed this pm check because it seems to me that it was used when we were auto detecting the package manager. Now that we offer a defined list of package managers it seems redundant to me.